threeML/plugins/OGIPLike.py

```diff
@@ -718,10 +721,35 @@ def get_simulated_dataset(self, new_name=None):
                 new_ogip_like._mask = original_mask
                 new_ogip_like._apply_mask_to_original_vectors()
 
+            # We want to store the simulated parameters so that the user
+            # can recall them
+
+            new_ogip_like._simulation_storage = copy.deepcopy(self._like_model)
```

Comment:
Astromodels provides the function clone_model for this. Can you use that instead of deepcopy?